packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -210,6 +211,11 @@ class Scrollable extends StatefulWidget {
   /// Determined by the [axisDirection].
   Axis get axis => axisDirectionToAxis(axisDirection);
 
+  /// Whether the scrollable scrolls in the reading direction.
+  ///
+  /// Defaults to false.
+  final bool reverse;
```

Comment:
Isn't this redundant with `axisDirection`? It seems like the fix should just be to check that when computing target offsets.

Reply:
correct me if I'm wrong, but `axisDirection` could be either `Axis.horizontal` or `vertical`, but it doesn't take into account if the axis is "reversed", so that's why `Scrollable` should be aware of this property

Reply:
See https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/widgets/scrollable.dart#L92

Reply:
@stuartmorgan I've used `axisDirection` to fix the issue